content/reference-rippled.md

```diff
@@ -737,6 +741,181 @@ The response follows the [standard format](#response-formatting), with a success
 
 
 
+## account_channels
+[[Source]<br>](https://github.com/ripple/rippled/blob/release/src/ripple/rpc/handlers/AccountChannels.cpp "Source")
+
+_(Requires the [PayChan amendment](concept-amendments.html#paychan) to be enabled. [New in: rippled 0.33.0][])_
+
+The `account_channels` method returns information about an account's Payment Channels. This includes only channels where the specified account is the source, not the destination. All information retrieved is relative to a particular version of the ledger.
+
+#### Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 1,
+  "command": "account_channels",
+  "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+  "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+  "ledger_index": "validated"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "account_channels",
+    "params": [{
+        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+        "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+        "ledger_index": "validated"
+    }]
+}
+```
+
+*Commandline*
+
+```bash
+#Syntax: account_channels <account> [<destination_account>] [<ledger>]
+rippled account_channels rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn validated
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The request includes the following parameters:
+
+| Field                 | Type                                       | Description |
+|:----------------------|:-------------------------------------------|:--------|
+| `account`             | String                                     | A unique identifier for the account whose channels to retrieve. Typically this is the account's [Address][]. |
```

Comment:
Well, technically neither my example nor yours are sentences; they're merely noun phrases. That said, although I'm confident my original phrasing is linguistically valid, I can agree it's not the clearest. How's this for a better restructuring:

> The unique identifier of an account, typically the account's Address. The request returns channels where this account is the channel's source.